src/js/intercom.ts

```diff
@@ -30,6 +33,6 @@ export function trackEvent(event: string) {
 export function update(options?: any) {
   const intercom = (window as any).Intercom;
   if (intercom) {
-    intercom('update', options);
+    intercom('update', { email: loggedInUserEmail, ...options });
```

Comment:
Some documentation says that it's fine to call 'update' without any parameters, so technically it shouldn't matter if it's called with an undefined email address. But I can add the check.